extensions/data-workspace/src/dialogs/openExistingDialog.ts

```diff
@@ -94,88 +70,35 @@ export class OpenExistingDialog extends DialogBase {
-						.withAdditionalProperties({ selectedTarget: 'workspace' })
-						.send();
-
-					// show git output channel
-					vscode.commands.executeCommand('git.showOutput');
-					// after this executes, the git extension will show a popup asking if you want to enter the workspace
-					await vscode.commands.executeCommand('git.clone', (<azdataType.InputBoxComponent>this.gitRepoTextBoxComponent?.component).value, this.localClonePathTextBox!.value);
-				} else {
-					await this.workspaceService.enterWorkspace(vscode.Uri.file(this.filePathTextBox!.value!));
-				}
-			} else {
-				// save datapoint now because it'll get set to new value during validateWorkspace()
-				const telemetryProps: any = { hasWorkspaceOpen: (vscode.workspace.workspaceFile !== undefined).toString() };
-
-				const validateWorkspace = await this.workspaceService.validateWorkspace();
-				let addProjectsPromise: Promise<void>;
-
-				if (this.remoteGitRepoRadioButton!.checked) {
-					TelemetryReporter.createActionEvent(TelemetryViews.OpenExistingDialog, TelemetryActions.GitClone)
-						.withAdditionalProperties({ selectedTarget: 'project' })
-						.send();
-
-					addProjectsPromise = this.workspaceService.gitCloneProject((<azdataType.InputBoxComponent>this.gitRepoTextBoxComponent?.component).value!, this.localClonePathTextBox!.value!, vscode.Uri.file(this.workspaceInputBox!.value!));
-				} else {
-					if (validateWorkspace) {
-						telemetryProps.workspaceProjectRelativity = calculateRelativity(this.filePathTextBox!.value!, this.workspaceInputBox!.value!);
-						telemetryProps.cancelled = 'false';
-						addProjectsPromise = this.workspaceService.addProjectsToWorkspace([vscode.Uri.file(this.filePathTextBox!.value!)], vscode.Uri.file(this.workspaceInputBox!.value!));
-					} else {
-						telemetryProps.workspaceProjectRelativity = 'none';
-						telemetryProps.cancelled = 'true';
-						addProjectsPromise = this.workspaceService.addProjectsToWorkspace([vscode.Uri.file(this.filePathTextBox!.value!)], vscode.Uri.file(this.workspaceInputBox!.value!));
-					}
-				}
-
-				TelemetryReporter.createActionEvent(TelemetryViews.OpenExistingDialog, TelemetryActions.OpeningProject)
-					.withAdditionalProperties(telemetryProps)
+			let addProjectsPromise: Promise<void>;
+
+			if (this.remoteGitRepoRadioButton!.checked) {
+				TelemetryReporter.createActionEvent(TelemetryViews.OpenExistingDialog, TelemetryActions.GitClone)
+					.withAdditionalProperties({ selectedTarget: 'project' })
 					.send();
 
-				await addProjectsPromise;
+				addProjectsPromise = this.workspaceService.gitCloneProject((<azdataType.InputBoxComponent>this.gitRepoTextBoxComponent?.component).value!, this.localClonePathTextBox!.value!);
+			} else {
+				telemetryProps.workspaceProjectRelativity = '';// TODO figure this out calculateRelativity(this.filePathTextBox!.value!, this.workspaceInputBox!.value!);
```

Comment:
not sure if we will still care about workspace project relativity if the workspace can be an untitled workspace (temp location)

Reply:
Remove all relativity telemetry per offline discussion